c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -1207,28 +1213,22 @@ class DeviceCachingAllocator {
 
       if (already_split && !block->expandable_segment_) {
         // An already-split inactive block is being shrunk by size bytes.
-        update_stat_array(
-            stats.inactive_split_bytes,
-            -static_cast<std::int64_t>(block->size),
-            params.stat_types);
+        decrease_stat_array(
+            stats.inactive_split_bytes, block->size, params.stat_types);
       } else if (!block->expandable_segment_) {
         // A new split inactive block is being created from a previously unsplit
         // block, size remaining->size bytes.
         for_each_selected_stat_type(params.stat_types, [&](size_t stat_type) {
-          update_stat(
-              stats.inactive_split_bytes[stat_type],
-              static_cast<std::int64_t>(remaining->size));
-          update_stat(stats.inactive_split[stat_type], 1);
+          increase_stat(stats.inactive_split_bytes[stat_type], remaining->size);
```

Comment:
Interesting that decrease has an _array version for this specific use case but not increase haha

Reply:
Increment and decrement are not symmetrical..